Automattic/woocommerce-payments
# Add advance filters on deposits page.

Implements #1545 

#### Changes proposed in this Pull Request
Added support for Advance filters - Date and status for deposits page.








#### Testing instructions
Need to be tested with 875-gh-Automattic/woocommerce-payments-server (functional, but yet to add tests)
- checkout to `add/1545-adv-filters-deposits`
- visit wcpay->deposits page.
- you should see advanced filters dropdown.
- ensure that selecting date and status filters give the desired result.






-------------------
